storage/src/vespa/storage/persistence/filestorage/filestormanager.cpp

```diff
@@ -114,7 +114,7 @@ FileStorManager::configure(std::unique_ptr<vespa::config::content::StorFilestorC
         size_t numThreads = _config->numThreads;
         _metrics->initDiskMetrics(_disks.size(), _component.getLoadTypes()->getMetricLoadTypes(), numThreads);
 
-        _filestorHandler.reset(new FileStorHandler(std::min(1ul, numThreads), *this, *_metrics, _partitions, _compReg));
+        _filestorHandler.reset(new FileStorHandler(std::min(numThreads, numThreads), *this, *_metrics, _partitions, _compReg));
```

Comment:
Remove `std::min` in favor of just `numThreads`?

Reply:
When I merge it it will be std::min(1ul, numThreads) to ensure we get same scaling as we do today. I will then increase to 2,4, 6 before letting it be numThreads in order to see any impact in our performance tests.